pkg/api/types.go

```diff
@@ -1993,6 +1993,8 @@ const (
 	NodeOutOfDisk NodeConditionType = "OutOfDisk"
 	// NodeMemoryPressure means the kubelet is under pressure due to insufficient available memory.
 	NodeMemoryPressure NodeConditionType = "MemoryPressure"
+	// NodeNetworkingReady means that network for the node is correctly configured.
+	NodeNetworkingReady NodeConditionType = "NetworkingReady"
```

Comment:
Except for NodeReady, where "True" means that everything is OK. I'm not sure about this conventions. @bgrant0607